### DISCUSSION TITLE
Support for qwik

### DISCUSSION THREAD

**Discussion by zanettin** [nxext/nx-extensions#818] [Ideas]
Hi there

Just curious if there are any plans to add qwik from builder.io to the list of supported frameworks somewhen or not. From my personal perspective this is since a long time a framework which brings the whole JS ecosystem a big step forward.
thanks for your feedback and votes if you feel like i do 😂

**Comment by DominikPieper** [comment]
I’m not sure yet if we should add it or not. Qwik is something on my list for interesting technologies since some time
